src/main/cpp/Controls.cpp

```diff
@@ -75,6 +75,16 @@ bool Controls::rJoyTriggerPressed()
     return rJoy_.GetTriggerPressed();
 }
 
+// Detects if button on the left side of the right joystick is pressed
+bool Controls::rJoyAltPressed() {
+    return rJoy_.GetRawButton(7);
```

Comment:
Please don't modify the Controls.cpp file; add a new method for this or something. Also we need to decide on a new button for the cube grabber.

Reply:
Sure, where should this new method go?

Reply:
I talked with the kyles and we're just switching button arangements, so I've already got that set up.